composer.json

```
@@ -0,0 +1,42 @@
+{
+    "name": "rubaxa/sortable",
```

```suggestion
    "name": "SortableJS/Sortable",
```

Does this change affect those in PHP land?